main.py

```diff
@@ -0,0 +1,176 @@
+import sys
+from csv import reader
+import datetime
+from math import ceil
+import os
+from statistics import mean
+
+
+def main(argv):
+    months = [" ", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+
+    length = len(sys.argv)
+    option = sys.argv[length - 2]
+    date = sys.argv[length - 1]
+    print(date)
+    if option == "-e":
+
+        date = sys.argv[length - 1]
+        year_month = datetime.datetime.strptime(date, "%Y")
+        year = str(year_month.year)
+        month = year
+        file_parsing(year, month, option)
+
+    elif option == "-a":
+        year_month = datetime.datetime.strptime(date, "%Y/%m")
+        year = str(year_month.year)
+        month = year_month.month
+        month_name = months[month]
+        file_parsing(year, month_name, option)
+
+    elif option == "-c":
+
+        year_month = datetime.datetime.strptime(date, "%Y/%m")
+        year = str(year_month.year)
+        month = year_month.month
+        month_name = months[month]
+        file_parsing(year, month_name, option)
+
+
+def get_first_second_temprature(dict1, dict2, month, year):
```

Comment:
As per Clean Code, a function having more than 2 params requires some serious justification. We can't simply increase number of params to a function, Ideally it should be one.